src/nn.h

```diff
@@ -342,6 +342,18 @@ NN_EXPORT  struct nn_cmsghdr *nn_cmsg_nxthdr_ (
 /*  Send/recv options.                                                        */
 #define NN_DONTWAIT 1
 
+/*  Global options.                                                           */
+#define NN_FORK_STRATEGY 1
+
+/*  Fork strategies.                                                          */
+/*  ABI warning: do not reorder, add new entries before NN_FORK_MAX_.         */
+enum nn_fork_strategy_kind {
+    NN_FORK_NONE = 1,
+    NN_FORK_RESET,
+
+    NN_FORK_MAX_
+};
+
```

Comment:
I did this to avoid incrementing the value of NN_FORK_MAX_ if another value was to be added, but I can use macro definitions instead.

I'm curious, though -- what is the rationale of not using enums originally? I was led to believe that integral constants should generally be represented as enums because of the type information they convey, but I certainly am not aware of everything that led to these design decisions.